src/lib/sensor_calibration/Accelerometer.cpp

```diff
@@ -63,6 +64,26 @@ void Accelerometer::set_device_id(uint32_t device_id)
 	}
 }
 
+void Accelerometer::set_external(bool external)
+{
+	// update priority default appropriately if not set
+	if (_calibration_index < 0) {
```

Comment:
Note, this is currently a duplicate of Magnetometer and can be refactored later.